docs/getting_started/hopsworksai/guides/getting_started.rst

```diff
@@ -0,0 +1,261 @@
+
+In the overview of your IAM user, select *Create access key*:
+
+.. _access-keys-aws-step-2.png: ../../../_images/access-keys-aws-step-2.png
+.. figure:: ../../../imgs/hopsworksai/access-keys-aws-step-2.png
+    :alt: Configuring the access key on AWS step 2
+    :target: `access-keys-aws-step-2.png`_
+    :align: center
+    :figclass: align-center
+
+Copy the *Access Key ID* and the *Secret Access Key*:
+
+.. _access-keys-aws-step-3.png: ../../../_images/access-keys-aws-step-3.png
+.. figure:: ../../../imgs/hopsworksai/access-keys-aws-step-3.png
+    :alt: Configuring the access key on AWS step 3
+    :target: `access-keys-aws-step-3.png`_
+    :align: center
+    :figclass: align-center
+
+Paste the *Access Key ID* and the *Secret Access Key* into Hopsworks.ai and select *Configure*:
+
+.. _save-access-key.png: ../../../_images/save-access-key.png
+.. figure:: ../../../imgs/hopsworksai/save-access-key.png
+    :alt: Saving the access key pair
+    :target: `save-access-key.png`_
+    :align: center
+    :figclass: align-center
+
+Step 2: Deploying a Hopsworks cluster
+-------------------------------------
+
+In Hopsworks.ai, select *Run a new instance*:
+
+.. _create-instance.png: ../../../_images/create-instance.png
+.. figure:: ../../../imgs/hopsworksai/create-instance.png
+    :alt: Create a Hopsworks cluster
+    :target: `create-instance.png`_
+    :align: center
+    :figclass: align-center
+
+Configure the instance by selecting the location, instance type and optionally the VPC,
+subnet and security group. Select *Deploy*.
+
+.. note::
+    We recommend that you always configure an SSH key to ensure you can troubleshoot the instance if necessary.
+
+.. _configure-instance.png: ../../../_images/configure-instance.png
+.. figure:: ../../../imgs/hopsworksai/configure-instance.png
+    :alt: Configuring the Hopsworks cluster
+    :target: `configure-instance.png`_
+    :align: center
+    :figclass: align-center
+
+The instance will start. This might take a couple of minutes:
+
+.. _booting.png: ../../../_images/booting.png
+.. figure:: ../../../imgs/hopsworksai/booting.png
+    :alt: Booting Hopsworks cluster
+    :target: `booting.png`_
+    :align: center
+    :figclass: align-center
+
+As soon as the instance has started, you will be able to log in to your new Hopsworks instance with the user name
+and password provided. We recommend that you change that password after your first login.
+You are also able to stop or terminate the instance.
+
+.. _running.png: ../../../_images/running.png
+.. figure:: ../../../imgs/hopsworksai/running.png
+    :alt: Running Hopsworks cluster
+    :target: `running.png`_
+    :align: center
+    :figclass: align-center
+
+Step 3: Exposing the Feature Store
+----------------------------------
+
+By default, only the Hopsworks UI is exposed by your Hopsworks instance. To integrate with external platforms and
+access APIs for services such as the Feature Store, you have to expose them.
+
```

Comment:
Should you not give a brief technical description of what this does in AWS. Like, 

In AWS, this will configure a Gateway to make X ports public to clients on external VPCs/subnetworks....

Reply:
Makes sense